### PR TITLE
Issue305 - Double tap on news comment

### PR DESCRIPTION
**Pull request by inderjeet-developer7**

### CODE REVIEW

File: app/lib/widgets/CommentView.dart [hunk 1]
```diff
@@ -22,162 +23,148 @@ class CommentView extends StatefulWidget {
...
                         ),
-                      ),
-                      Text(
-                        widget.comment,
-                        style: const TextStyle(
-                          color: Colors.white,
-                          fontSize: 16,
+                        Text(
+                          widget.commentModel.comment,
+                          style: const TextStyle(
```
Comment (gtalha07) [inline app/lib/widgets/CommentView.dart:75]: Please add theme constants in ```seperatedThemes.dart``` if reusable.

File: app/lib/widgets/ReplyView.dart [hunk 2]
```diff
@@ -37,11 +39,12 @@ class ReplyViewState extends State<ReplyView> {
                 crossAxisAlignment: CrossAxisAlignment.start,
                 children: [
                   Text(
-                    widget.name,
-                    style: TextStyle(color: widget.titleColor, fontSize: 16),
+                    widget.replyModel.name,
+                    style: TextStyle(
+                        color: widget.replyModel.titleColor, fontSize: 16),
```
Comment (gtalha07) [inline app/lib/widgets/ReplyView.dart:44]: Same here.

**Review by gtalha07 — APPROVED** [review]
LGTM!

File: app/lib/widgets/CommentView.dart (hunk 1 above)
Comment (inderjeet-developer77) [reply app/lib/widgets/CommentView.dart:75]: we cannot as right now I am passing color from the previous screen.. once API is integrated... we will set colors randomly

File: app/lib/widgets/ReplyView.dart (hunk 2 above)
Comment (inderjeet-developer77) [reply app/lib/widgets/ReplyView.dart:44]: same as above

**Pull request merged**